alertmanager/multitenant.go

```diff
@@ -21,12 +22,55 @@ import (
 	configs "github.com/weaveworks/cortex/configs/client"
 	"github.com/weaveworks/cortex/util"
 	"github.com/weaveworks/mesh"
+	"strings"
 )
 
 const (
 	// Backoff for loading initial configuration set.
 	minBackoff = 100 * time.Millisecond
 	maxBackoff = 2 * time.Second
+
+	statusPage = `
+<!doctype html>
+<html>
+	<head><title>Cortex Alertmanager Status</title></head>
+	<body>
+		<h1>Cortex Alertmanager Status</h1>
+        <h2>Mesh router</h2>
+        <dl>
+          <dt>Protocol</dt>
+          <dd>{{.Protocol}}
+{{if eq .ProtocolMinVersion .ProtocolMaxVersion}}
+{{.ProtocolMaxVersion}}
+{{else}}
+{{.ProtocolMinVersion}}..{{.ProtocolMaxVersion}}
+{{end}}
```

Comment:
I'd indent this block as well. If you're worried about extra whitespace, you can gobble it up via `-}}` and `{{-`. Quote from docs:

"However, to aid in formatting template source code, if an action's left delimiter (by default "{{") is followed immediately by a minus sign and ASCII space character ("{{- "), all trailing white space is trimmed from the immediately preceding text. Similarly, if the right delimiter ("}}") is preceded by a space and minus sign (" -}}"), all leading white space is trimmed from the immediately following text."